DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py

```diff
@@ -390,11 +407,80 @@ def ftsTransfer(self):
     # Just in case some transfers could not be scheduled, try them with RM
     return self.dmTransfer(fromFTS=True)
 
+  def _checkExistingFTS3Operations(self):
+    """
+       TL;DR: check if there are ongoing FTS3Operation for the current RMS Operation
```

Comment:
The first line of the docstring is by convention a TL;DR

Reply:
OK, changed

Reply:
Oops